appium/webdriver/webdriver.py

```diff
@@ -799,15 +795,24 @@ def pull_folder(self, path):
         }
         return self.execute(Command.PULL_FOLDER, data)['value']
 
-    def push_file(self, path, base64data):
-        """Puts the data, encoded as Base64, in the file specified as `path`.
+    def push_file(self, destination_path, base64data=None, source_path=None):
+        """Puts the data from the file at `source_path`, encoded as Base64, in the file specified as `path`.
 
+        Specify either `base64data` or `source_path`, if both specified default to `source_path`
         :Args:
-         - path - the path on the device
+         - destination_path - the path on the device
```

Comment:
I'd rather add more advanced docstrinmg here, since several formatting options are available there. Check https://github.com/appium/java-client/blob/f527be18735f8a60b4f6244f03a1b9954c23a60f/src/main/java/io/appium/java_client/InteractsWithFiles.java#L44

and 

https://github.com/appium/appium-xcuitest-driver/blob/31059c4169516098d2b64a328197a53000ecd360/lib/commands/file-movement.js#L186

for more details

Reply:
Sure I update the formatting to the reStructured text format, let me know if that one's good. Originally I was just going along with the format I saw in the rest of the file to stay consistent but now I'm noticing some other functions use this format as well.